api/src/db.js

```diff
@@ -63,20 +63,21 @@ if (UNIT_TEST_ENV) {
 } else {
   const fetch = (url, opts) => {
     // Adding audit flag (haproxy) Service that made the request initially.
-    opts.headers.set('X-Medic-Service', 'api');
+    opts.headers.set('X-CHT-Service', 'api');
```

Comment:
Be sure to update the haproxy.cfg to match: https://github.com/medic/medic-infrastructure/blob/152a9230a4573a7551610c05a21225b9155dfd48/images/haproxy/haproxy.cfg#L35

Reply:
Ah yea ... I can revert or we can convene and decide whether it's of any importance or not. @Hareet what do you think?

Reply:
I've reverted this. We can decide on this later. Thanks for catching it!